lib/capybara/node/finders.rb

```diff
@@ -31,12 +35,14 @@ module Finders
       #                                       * false - only find elements whose centerpoint is in the viewport and is not obscured by other non-descendant elements.
       #   @option options [String, Regexp]  id           Only find elements with an id that matches the value passed
       #   @option options [String, Array<String>, Regexp] class  Only find elements with matching class/classes.
-      #                                            + Absence of a class can be checked by prefixing the class name with !
-      #                                            + If you need to check for existence of a class name that starts with ! then prefix with !!
-      #                                            + ```class:['a', '!b', '!!!c'] # limit to elements with class 'a' and '!c' but not class 'b'```
+      #                                            * Absence of a class can be checked by prefixing the class name with `!`
+      #                                            * If you need to check for existence of a class name that starts with `!` then prefix with `!!`
+      #
+      #                                                class:['a', '!b', '!!!c'] # limit to elements with class 'a' and '!c' but not class 'b'
+      #
       #   @option options [String, Regexp, Hash] style  Only find elements with matching style. String and Regexp will be checked against text of the elements `style` attribute, while a Hash will be compared against the elements full style
-      #   @option options [Boolean] exact            Control whether `is` expressions in the given XPath match exactly or partially
-      # @option options [Symbol] match        The matching strategy to use.
+      #   @option options [Boolean] exact            Control whether `is` expressions in the given XPath match exactly or partially. Defaults to {Capybara.configure exact}.
```

Comment:
[note] The `Capybara.configure exact` doc is missing now. I don't know how to write the doc, sorry. 🙇 

https://github.com/teamcapybara/capybara/blob/4bc8101afc664d15ddafc8ea11ec262fd9258365/lib/capybara.rb#L102

Reply:
I've added `exact` to the configure documentation

Reply:
Thank you!